crates/esthri/src/presign/upload.rs

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2022 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+use std::{os::unix::prelude::MetadataExt, path::Path, time::Duration};
```

Comment:
```suggestion
use std::{path::Path, time::Duration};
```
think this unix specific - can delete this if we use len() - (*not actually sure the difference tho the docs sounds like they do exactly the same thing*)

Reply:
Good pick up, changed